packages/rancher-pushprox/charts/templates/pushprox-clients.yaml

```diff
@@ -1,18 +1,28 @@
 {{- if .Values.clients }}{{- if .Values.clients.enabled }}
 apiVersion: apps/v1
+{{- if .Values.clients.deployment.enabled }}
+kind: Deployment
+{{- else }}
 kind: DaemonSet
+{{- end }}
 metadata:
   name: {{ template "pushProxy.client.name" . }}
   namespace: {{ template "pushprox.namespace" . }}
   labels: {{ include "pushProxy.client.labels" . | nindent 4 }}
     pushprox-exporter: "client"
 spec:
+  {{- if .Values.clients.deployment.enabled }}
+  replicas: {{ .Values.clients.deployment.replicas }}
+  {{- end }}
   selector:
     matchLabels: {{ include "pushProxy.client.labels" . | nindent 6 }}
   template:
     metadata:
       labels: {{ include "pushProxy.client.labels" . | nindent 8 }}
     spec:
+      {{- if .Values.clients.deployment.enabled }}
+      affinity: {{ toYaml .Values.clients.deployment.affinity | nindent 8 }}
+      {{- end }}
```

Comment:
affinity should be under `.Values.clients.affinity`, re: https://github.com/rancher/charts/pull/1227#discussion_r645795496

Reply:
fixed.